src/tools/std23/views/test/stride.test.cpp

```diff
@@ -0,0 +1,139 @@
+// include Catch2
+#include <catch2/catch_test_macros.hpp>
```

Comment:
A valid point. And it did not work as it should have in this state. I actually added a test case later on to verify if these would work in our use cases in ENDFtk, see #27 for that test. I found multiple bugs because of it which are also corrected in that PR.

Reply:
We also should add more tests on the iterator operations (some of these are used under the hood in the view_interface functions but they most likely do not cover everything).